addon_service/common/network.py

```diff
@@ -60,7 +63,7 @@ def __init__(
         *,
         client_session: aiohttp.ClientSession,
         prefix_url: str,
-        account: db.AuthorizedStorageAccount,
+        account: "AuthorizedStorageAccount",
```

Comment:
I *think* that newer versions of python don't require the quotes -- though it may still require a `from __future__ import annotations` to omit them?

Reply:
ohh, for some reason i thought `from __future__ import annotations` was for use in older versions to get current behavior (the _past_'s future), but yeah that's exactly what should do here